go-controller/pkg/libovsdb/util/acl.go

```diff
@@ -136,6 +136,12 @@ func getACLMatch(portGroupName, match string, aclDir aclDirection) string {
 	return aclMatch
 }
 
+// ACL logging severity levels
```

Comment:
note to self: moved here from go-controller/pkg/ovn/ovn.go

Reply:
yup, was better to move it here..